web/client/plugins/print/index.js

```diff
@@ -128,7 +128,7 @@ const PrintSubmit = connect((state) => ({
 }))(require('../../components/print/PrintSubmit'));
 
 const PrintPreview = connect((state) => ({
-    url: state.print && state.print.pdfUrl,
+    url: state.print && ConfigUtils.getProxyUrl(state.print.pdfUrl),
```

Comment:
You should use the proxied url only if it's not from the sameOrigin. Look here: https://github.com/geosolutions-it/MapStore2/blob/master/web/client/libs/ajax.js#L155

In addition: ConfigUtils.getProxyUrl(...) can return an object (look at the code linked above), you should take care of this. The best option would be to extract the code in ajax.js that handles proxyUrl, sameOrigin, useCORS and put it in a utility method.